src/prefect/events/schemas.py

```diff
@@ -239,8 +282,8 @@ def enforce_minimum_within(
 
     @root_validator(skip_on_failure=True)
     def enforce_minimum_within_for_proactive_triggers(cls, values: Dict[str, Any]):
-        posture: Optional[Posture] = values.get("posture")
-        within: Optional[timedelta] = values.get("within")
+        posture: Posture | None = values.get("posture")
```

Comment:
CI has probably already told you this, but gotta use `Optional` for 3.8 support.

Reply:
Woops it did NOT tell me that :doh: thank you!